.github/workflows/report-robot-results.yml

```diff
@@ -0,0 +1,49 @@
+name: "Collect Robot Results"
```

Comment:
Do we need to remove `Download - Robot results` the step from the `Build & Test` worfklow

Reply:
yes in a dedicated PR, I want to try it out before removing it - chicken eg problem, the new workflow_run can only be used when it is available in the default branch => `develop`.